.github/workflows/build_pull_request.yml

```diff
@@ -1,23 +1,15 @@
-name: PR Build
+name: Build pull request
 
 on:
   workflow_dispatch:
     inputs:
-      # Flutter
-      flutter-branch:
```

Comment:
I removed this because it didn't look like this was necessary. We only really use stable Flutter branch